### PR TITLE
feat: Wrap Native Token

### PR DESCRIPTION
**Pull request by Picodes**

### CODE REVIEW

File: ERC20_LIST.json [hunk 1]
```diff
@@ -648,6 +679,25 @@
         "useInSwap": false,
         "hasPermit": false,
         "logoURI": "https://raw.githubusercontent.com/AngleProtocol/angle-assets/main/0_tokens/ANGLE-token/cross-chain/ANGLE-avax.svg"
+      },
+      "0xB31f66AA3C1e785363F0875A1B74E27b85FD66c7": {
+        "address": "0xB31f66AA3C1e785363F0875A1B74E27b85FD66c7",
+        "name": "Wrapped AVAX",
+        "decimals": 18,
+        "symbol": "AVAX",
```
Comment (greedythib) [inline ERC20_LIST.json:687]: no underlying then?

Comment (Picodes) [reply ERC20_LIST.json:687]: it should be 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE

File: ERC20_LIST.json [hunk 2]
```diff
@@ -628,6 +641,24 @@
...
+        "address": "0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c",
+        "name": "Wrapped BNB",
+        "decimals": 18,
+        "symbol": "WBNB",
+        "useInSwap": false,
+        "hasPermit": false,
+        "logoURI": "https://raw.githubusercontent.com/AngleProtocol/angle-token-list/main/src/assets/tokens/BNB.svg"
+      },
+      "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE": {
+        "address": "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE",
```
[the remark this answers is not in the document]
Comment (Picodes) [reply ERC20_LIST.json:655]: Object.values(tokenList)?.filter((t) => t.wrappingMethod === 'Wrap Native')?.[0]

File: ERC20_LIST.json (hunk 1 above)
Comment (Picodes) [reply ERC20_LIST.json:687]: I added it under the entry 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE

**Review by greedythib — APPROVED** [review]

**Pull request merged**